sdk/src/test/scala/com/horizen/account/api/http/AccountNodeViewUtilMocks.scala

```diff
@@ -0,0 +1,225 @@
+package com.horizen.account.api.http
+
+import com.horizen.account.node.{AccountNodeView, NodeAccountHistory, NodeAccountMemoryPool}
+import com.horizen.account.transaction.EthereumTransaction
+import com.horizen.api.http.SidechainApiMockConfiguration
+import com.horizen.fixtures._
+import com.horizen.node.{NodeStateBase, NodeWalletBase}
+import org.mockito.Mockito
+import org.scalatestplus.mockito.MockitoSugar
+import org.web3j.crypto.{ECKeyPair, Keys, RawTransaction, Sign, SignedRawTransaction}
+
+import java.nio.charset.StandardCharsets
+import java.util
+
+class AccountNodeViewUtilMocks extends MockitoSugar with BoxFixture with CompanionsFixture {
+
+  //  val sidechainTransactionsCompanion: SidechainTransactionsCompanion = getDefaultTransactionsCompanion
```

Comment:
Actually I left those comments because I imagined that the code would have been used in the future, when the implementation had been further improved. But probably it is better to clean up.